src/Compute/Compute/ChangeLog.md

```diff
@@ -20,6 +20,7 @@
 
 -->
 ## Upcoming Release
+* Updated `New-AzVM`, `New-AzVmss`, `Update-AzVM`, and `Update-AzVmss` to pass `Standard` as an input of `-SecurityType` parameter.
```

Comment:
Looks like there are no user facing changes with these updates, but if there are then update at least 1 md file.

Reply:
no change on customer side to be reflected in md file